app/views/classified_listings/index.html.erb

```diff
@@ -36,7 +36,7 @@
 
     <div class="classified-filters">
     <div class="classified-filters-categories">
-      <a href="<%= classified_listings_path %>" class="<%= "selected" if params[:category].blank? %> data-no-instant="true">all</a>
```

Comment:
this has come back from the side :-D the previous version was correct, the one without the spaces in the attribute

Reply:
ugh??? i have no idea why is this included in my PR... git is weird... reverting that..

Reply:
reverted.